packages/saved-views-client/src/models/Group.ts

```diff
@@ -8,5 +8,6 @@ export interface Group {
   id: string;
   displayName: string;
   shared: boolean;
+  readOnly: boolean;
```

Comment:
I'm concerned that a group might not contain `readOnly` property if it was created before this update. Is that plausible? If so, this property should be marked optional.

Reply:
After applying TypeORM migrations in DEV and QA all groups created prior have the readOnly property set to false. 
However in PROD groups do not have the readOnly property, as we haven't released and migrated yet

Reply:
By default the client targets PROD environment so I think it's best to wait for the server side change to be propagated first.